lib/static/modules/actions/sort-tests.ts

```diff
@@ -0,0 +1,19 @@
+import actionNames from '@/static/modules/action-names';
+import {Action} from '@/static/modules/actions/types';
+import {SortDirection} from '@/static/new-ui/types/store';
+
+type SetCurrentSortByExpressionAction = Action<typeof actionNames.SORT_TESTS_SET_CURRENT_EXPRESSION, {
+    expressionIds: string[];
+}>;
+export const setCurrentSortByExpression = (payload: SetCurrentSortByExpressionAction['payload']): SetCurrentSortByExpressionAction =>
+    ({type: actionNames.SORT_TESTS_SET_CURRENT_EXPRESSION, payload});
+
+type SetSortByDirectionAction = Action<typeof actionNames.SORT_TESTS_SET_DIRECTION, {
```

Comment:
I thought it's more convenient this way in actions specifically, because we have action type and action creator close to each other. We didn't have typescript actions before, so I thought it wouldn't break any conventions